source/scss/blocks/rules.scss

```
@@ -180,16 +200,90 @@
   p {
     margin: 0;
 
+    @include screenActive() {
+      animation-name: showRulesItemText;
+      animation-duration: 0.5s;
+      animation-fill-mode: both;
+      animation-timing-function: cubic-bezier(0.49, 0.4, 0.15, 1.01);
+      animation-delay: calc(var(--rules-item-delay) + 0.15s);
+    }
+
+    @keyframes showRulesItemText {
+      from {
+        opacity: 0;
+        transform: translate3d(1em, 0, 0);
+      }
+
+      to {
+        opacity: 1;
+        transform: translate3d(0, 0, 0);
+      }
+    }
+
     @media (max-width: $tablet) and (orientation: portrait) {
       min-height: calc(1.4rem * 2.2);
     }
   }
+
+  &:nth-child(1) {
+    --rules-item-index: 0;
+  }
+
+  &:nth-child(2) {
+    --rules-item-index: 1;
+  }
+
+  &:nth-child(3) {
+    --rules-item-index: 2;
+  }
+
+  &:nth-child(4) {
+    --rules-item-index: 3;
+  }
```

Здесь можно немного сократить, используя цикл @for:
```suggestion
  @for $i from 1 through 5 {
    &:nth-child(#{$i}) {
      --rules-item-index: #{$i - 1};
    }
  }
```

---

Спасибо! Сократила)